docs/application/tizen-studio/release-notes/3-3-release-notes.md

```diff
@@ -0,0 +1,66 @@
+# Tizen Studio 3.3 Release Notes
+
+-   Release Date: July 26, 2019
+
+
+## IDE and Tools
+
+
+### Fixed Bugs
+-   Device Manager
+	-   Fixed the issue with logs not shown for Raspberry Pi board.
+-   IDE
+	-   Fixed the log view to work with TV devices in Tizen IDE.
+	-   App signing command incorrectly asking for Author and Distributor passwords has been fixed.
```

Comment:
To maintain consistency with other Release Notes (has/have been), please rephrase to:
The issue of app signing command incorrectly asking for the author and the distributor passwords has been fixed.
Please verify.